CHANGES/1322.bugfix

```diff
@@ -0,0 +1 @@
+Fix endpoint repositories/ansible/ansible/{repository_pk}/versions/{number}/rebuild_metadata/
```

Comment:
Please write in past tense. Also think about the end user reading this. How about:
```suggestion
Fixed a 500 server error in the ``repositories/ansible/ansible/{repository_pk}/versions/{number}/rebuild_metadata/`` endpoint.
```

Reply:
sounds better, thanks :+1: